app/about/index.html

```diff
@@ -1,8 +1,5 @@
 ---
 layout: default
-title: DEVit Conference - Thessaloniki 20 May 2016
```

Comment:
@vkarampinis we need those to override the defaults in each page for better SEO. We should add them again with the right content.